rslidar_pointcloud/src/rawdata.cc

```diff
@@ -88,19 +88,30 @@ void RawData::loadConfigFile(ros::NodeHandle node, ros::NodeHandle private_nh)
 
   ROS_INFO_STREAM("[cloud][rawdata] initialize resolution type: " << (dis_resolution_mode_?"1 cm":"0.5 cm") << ", intensity mode: " << intensity_mode_);
 
-  private_nh.param<float>("start_clip_angle_deg", start_clip_angle_, 0.F);
-  private_nh.param<float>("end_clip_angle_deg", end_clip_angle_, 360.F);
+  std::vector<float> start_clip_angles;
+  std::vector<float> end_clip_angles;
+  private_nh.getParam("start_clip_angles", start_clip_angles);
+  private_nh.getParam("end_clip_angles", end_clip_angles);
 
-  assert(std::abs(start_clip_angle_) <= 360.F);
-  assert(std::abs(end_clip_angle_) <= 360.F);
+  assert(start_clip_angles.size() == end_clip_angles.size());
```

Comment:
That's useless. Runtime check only, since you don't have control over params, but the driver is always built with `Release` which should set `NDEBUG`